port/cpl_http.cpp

```diff
@@ -1150,12 +1150,15 @@ int CPLHTTPPopFetchCallback(void)
  *     and use the first one found as the CAINFO value (GDAL >= 2.1.3). The
  *     GDAL_CURL_CA_BUNDLE environment variable may also be used to set the
  *     CAINFO value in GDAL >= 3.2.</li>
- * <li>HTTP_VERSION=1.0/1.1/2/2TLS (GDAL >= 2.3). Specify HTTP version to use.
+ * <li>HTTP_VERSION=1.0/1.1/2/2TLS (GDAL >= 2.3)/2PRIOR_KNOWLEDGE (GDAL >= TBD).
```

Comment:
Would this be 3.10 or 4.0?

And do you have any nits about the phrasing and word wrapping?